include/UtfView/code_unit_view.hpp

```diff
@@ -0,0 +1,608 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+#ifndef UTFVIEW_CODE_UNIT_VIEW_HPP
+#define UTFVIEW_CODE_UNIT_VIEW_HPP
+
+#include <UtfView/detail/concepts.hpp>
+#include <boost/stl_interfaces/iterator_interface.hpp>
```

Comment:
No change needed here, but just a heads-up.  I've pulled iterator_interface from consideration for C++26, because I was to see what it looks like to do it using the new reflection language feature we'll get in 26.  Sine this paper is also probably targeting 29, this is maybe a non-issue; I fully intend to resurrect the iterator_interface paper for 29.

Reply:
We'll see how the scheduling goes, but I ideally was hoping to target 26 because P2871 and P2873 are targeting 26 and I wanted this to replace the removed `std::codecvt` facets. That's not a showstopper, though; if it turns out I have to wait for 29, I'll leave `iterator_interface` in, and if I do manage to target 26, I can take out `iterator_interface` at the expense of verbosity.